amd_openvx_extensions/amd_rpp/include/vx_ext_rpp.h

```diff
@@ -164,6 +164,7 @@ extern "C" SHARED_PUBLIC vx_node VX_API_CALL vxRppGlitch(vx_graph graph, vx_tens
 extern "C" SHARED_PUBLIC vx_node VX_API_CALL vxRppNoise(vx_graph graph, vx_tensor pSrc, vx_tensor pSrcRoi, vx_tensor pDst, vx_array pNoiseProb, vx_array pSaltProb, vx_array pNoiseValue, vx_array pSaltValue, vx_scalar seed, vx_scalar inputLayout, vx_scalar outputLayout, vx_scalar roiType);
 extern "C" SHARED_PUBLIC vx_node VX_API_CALL vxRppResizeCrop(vx_graph graph, vx_tensor pSrc, vx_tensor pSrcRoi,vx_tensor pCropTensor, vx_tensor pDst, vx_array pDstBatchWidth, vx_array pDstBatchHeight, vx_scalar inputLayout, vx_scalar outputLayout, vx_scalar roiType);
 extern "C" SHARED_PUBLIC vx_node VX_API_CALL vxRppSequenceRearrange(vx_graph graph,vx_tensor pSrc,vx_tensor pDst, vx_array pNewOrder, vx_scalar layout);
+extern "C" SHARED_PUBLIC vx_node VX_API_CALL vxExtrppNode_PreemphasisFilter(vx_graph graph, vx_tensor pSrc, vx_tensor pDst, vx_tensor srcSamplesSize, vx_tensor dstSamplesSize, vx_array preemphCoeff, vx_scalar borderType);
```

Comment:
we have been following below argument order for almost all of the augmentations
pSrc, pSrcDims/pSrcLength, pDst, pDstDims 

would suggest to change the order here as well to make it consistent
Also pointer names have been changed to have a prefix p, So we have something like pSrcSamples here

For pointers buffer having dims for input and output
I have named as pSrcLength - 1D audio length buffer
                           pSrcDims, pDstDims - 2D audio length buffer
please check it is better to have naming convention style and change

Reply:
Most of the args have pSrcRoi & pDstRoi for the already present image augmentations.
Example : In Warp Affine : `	 * \param [in] pSrc The input tensor in <tt>\ref VX_TYPE_UINT8</tt> or <tt>\ref VX_TYPE_FLOAT32</tt> or <tt>\ref VX_TYPE_FLOAT16</tt> or <tt>\ref VX_TYPE_INT8</tt> format data.
`

Reply:
Made the other changes